src/api/deploy-webshell/helpers/can-run-shell-in-environment.js

```diff
@@ -0,0 +1,33 @@
+import { config, environments } from '~/src/config'
+
+/**
+ * Is a user allowed to run a test suite in given environment.
+ *
+ * @param {string} environment
+ * @param {string[]} scope
+ * @param {string} adminGroup
+ * @return {boolean}
+ */
+export function canRunShellInEnvironment(
+  environment,
+  scope,
+  adminGroup = config.get('oidcAdminGroupId')
+) {
+  if (environment === environments.prod) {
+    // At some point we might want to provide a break-glass for this
+    return false
+  }
+
+  if (scope.includes(adminGroup)) {
+    return true
+  }
+
+  switch (environment) {
```

Comment:
I'd get rid of this switch. They're a little evil. You've covered the prod case above.

```
if (environment === environments.infraDev || environment === environments.management) {
   return false
}

return true

```